docs/content/interop.md

```diff
@@ -43,10 +43,8 @@ For further information on the settings available through the Unreal GDK toolbar
 
 ### Example
 
-The image below demonstrates how we set up the class `StarterProjectPlayerController` in the [Unreal GDK Starter Project](https://github.com/spatialos/UnrealGDKStarterProject). The `StarterProjectPlayerController` depends on the following include files:`StarterProjectPlayerController.h`,`Camera/CameraAnim.h`,`Camera/CameraShake.h`,`GameFramework/HUD.h`,`GameFramework/LocalMessage.h`, and `Particles/EmitterCameraLensEffectBase.h`.
-
-![Interop Code Generator Settings](../assets/screen_grabs/toolbar/ICGSettings.png)
-_Set up of the `StarterProjectPlayerController` class in the Starter Project_
+Identify the classes you which to replicate across Spatial, and tag each UClass with the `Spatial` keyword. The image below demonstrates how we set up the class `StarterProjectPlayerController` in the Unreal GDK Starter Project. 
+![Tagging Spatial in UClasses](../assets/screen_grabs/toolbar/spatial_interop_tagging.png)
```

Comment:
I can't see this but we usually use code snippets and examples in text in line with...

```
Here I am

```
This helps users to copy and paste it. It also makes it easier to fix and update.